syntax = "proto3";

package executor;

service Executor {
  rpc GetResult(Req) returns (Res) {}
}

message Req {
    int32 edition = 1;
    string code = 2;
}

message Res {
    string c_stdout_str = 1;
    string c_stderr_str = 2;
    int32 exit_code = 3;
    string stdout_str = 4;
    string stderr_str = 5;
}
